sass/layout/_sidebar-left.scss

```diff
@@ -30,7 +30,7 @@
     left: 0;
     height: 100%;
     z-index: 16;
-    border-right: 1px solid rgba(var(--center-channel-color-rgb), 0.2);
+    border-right: 1px solid var(--center-channel-color-08);
```

Comment:
Same in these other files. We shouldn't be reverting to the transparent theme variables.